packages/certificate-model/package.json

```diff
@@ -19,43 +19,17 @@
     "access": "public"
   },
   "private": false,
-  "main": "dist/index",
-  "browser": "dist.browser/index",
-  "types": "dist/index",
-  "files": [
-    "dist"
-  ],
+  "main": "src/index.js",
+  "types": "src/index.d.ts",
   "engines": {
     "node": ">=12",
     "npm": ">=6"
   },
   "scripts": {
```

Comment:
the command in scripts should be moved as targets in project.json except for `prepublishOnly` that can be removed.
In the end the package.json will only be used when installing the lib from npm.